### PR TITLE
Stones and Eggs everywhere

### PR DESCRIPTION
**Pull request by Ultima1990**
After the region they first show up in of course.

Did I miss any?

### CODE REVIEW

**Comment by RedSparr0w** [comment]
Sorry, looks like there's some code conflicts that need fixing before merge.

File: src/scripts/towns/Town.ts [hunk 1]
```diff
@@ -371,9 +371,6 @@ TownList['Cerulean Cave'] = new DungeonTown(
 const NewBarkTownShop = new TownShop([
     ItemList['Pokeball'],
 ]);
-const CherrygroveCityShop = new TownShop([
-    ItemList['Greatball'],
```
Comment (RedSparr0w) [inline src/scripts/towns/Town.ts:375]: Are these still available elsewhere in the region?

**Review by RedSparr0w — APPROVED** [review]
Thanks for the PR! 👍

File: src/scripts/towns/Town.ts (hunk 1 above)
Comment (RedSparr0w) [reply src/scripts/towns/Town.ts:375]: Nvm, found em 👍

Comment (Ultima1990) [reply src/scripts/towns/Town.ts:375]: Yes they are in Ecruteak.

**Pull request merged**